components/src/core/Drawer/DrawerFooter/DrawerFooter.tsx

```diff
@@ -76,9 +75,9 @@ const DrawerFooterRender: React.ForwardRefRenderFunction<unknown, DrawerFooterPr
     } = props;
     const { open: drawerOpen = true } = useDrawerContext();
     return (
-        <Root>
+        <Fragment>
```

Comment:
I think I would prefer to pass sx through to the Root in this situation.

You can add this as a prop:
````sx?: SxProps;```